contribution.md

```diff
@@ -0,0 +1,500 @@
+# Developing a component
+
+  This is a tutorial how to setup a project with Corporate-UI components with Scania-theme styling.
+
+  Corporate UI is a toolbox with **UI components** that makes it easier to use reusable and modular components for your application. For example Header or a footer are two components, but also buttons and dropdowns are also included.
+
+  Scania-Theme is a packages to contains the Scania specific styling for everyone of those components in Corporate UI toolbox. Every component has a unique styling with Scania look and feel.
+
+
+## Components
+
+ Header,footer or a button are all components in our Design System, the difference between them is how they technically are made.
+
+ `<c-header></c-header>` is a web component, while a `<button> Click Me </button>` is a regular HTML element that already exist in the HTML Standard. One thing easy thing to look at is the dash, for example "c-header", that indicates it is a web-component.
```

Comment:
"One easy thing to differentiate between those is by looking at the dash in the element name, for example...."

Reply:
Added